app/javascript/components/public_health/CustomExport.js

```diff
@@ -521,7 +521,9 @@ class CustomExport extends React.Component {
         </Modal>
         <ConfirmExport
           show={this.state.show_confirm_export_modal}
-          title={`Custom Export Format ${this.state.preset?.name ? `(${this.state.preset.name})` : ''}`}
+          exportType={'Custom Export Format'}
+          presetName={this.state.preset?.name}
+          // title={`Custom Export Format ${this.state.preset?.name ? `(${this.state.preset.name})` : ''}`}
```

Comment:
Please remove commented out line if not needed (which it doesn't appear to be anymore).

Reply:
done